umibukela/templates/partners.html

```diff
@@ -43,24 +43,14 @@
             <th>Province</th>
           </tr>
         </thead>
+{% for partner in partners %}
         <tr>
-          <td>Tshedza Development Project</td>
+          <td><a href="{% url 'partner' partner.slug %}">{{ partner.short_name }}</a></td>
+          <td><a href="{% url 'site-result' site_name='folweni' result_id='4' %}"></a></td>
           <td></td>
-          <td>Social Development</td>
-          <td>Gauteng</td>
```

Comment:
I'm not sure what the Gauteng is meant to be -- is that their focus area, or their address province? Can you double check with Jen?

Can we add sector to the models and have a many-to-one link to a partner? Otherwise I feel like we're not going to revisit this, now is a good time to do it.